pglast/stream.py

```diff
@@ -357,10 +361,22 @@ def print_node(self, node, is_name=False, is_symbol=False):
                 printer(node, self)
         self.separator()
 
+    def _is_pg_catalog_func(self, items):
+        return (
+            self.remove_pg_catalog_from_functions
+            and items.parent_attribute == 'funcname'
+            and len(items) > 1
+            and items[0].val.value == 'pg_catalog'
+            and items[1].val.value not in ['position']
```

Comment:
Please, add a comment here explaining the need for the check against `position`. Also, why use ``not in []`` instead of straight ``==``?

Reply:
I expect eventally to find another function that can't be used without the schema prefix.

Reply:
Ok then, I'd use a tuple instead of a list.